src/main/java/com/example/module2/StudentMain.java

```diff
@@ -3,122 +3,152 @@
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Date;
 import java.util.List;
 
 public class StudentMain {
     public static void main(String[] args) throws ParseException {
-        SimpleDateFormat format = new SimpleDateFormat("dd.MM.yyyy");
-
-        Student student = new Student("Doe", "Joe", "Jonovich");
-        Student student1 = new Student("Billigan", "Tommy", "Joevich", "+375 25 555 00 42");
-        Student student2 = new Student("Grib", "Olga", "Ol'govna", "+375 55 874 98 77");
-        Student student3 = new Student("Wick", "John", "Jordanovich", "+7 983 778 00 07");
-        Student student4 = new Student("Potter", "Harry", "Ivanovich", "+89 983 784 83 98");
-        Student student5 = new Student("Batty", "Roy", "Nexsus 6", "+42 424 242 42 42");
-
-        Address address = new Address("Singapure", "Kuara Lumupur",
-                18765, "Borrowman", 7897, 2);
-        Address address1 = new Address("Belarus", "Minsk",
-                220000, "Nezalejnosti", 4, 42);
-        Address address2 = new Address("Poland", "Wrzlav",
-                297000, "Klaus Pshche", 8, 192);
-        Address address3 = new Address("US", "Oklahoma",
-                987846, "Tompsom Johnes", 7, 11);
-        Address address4 = new Address("UK", "Belfast",
-                111000, "God Save Queen", 1, 1);
-        Address address5 = new Address("US", "NewNewYork",
-                544500, "Hopes", 98, 3);
-
-        String date = "01.03.2000";
-        String date1 = "20.02.1931";
-        String date2 = "11.11.1976";
-        String date3 = "01.01.1970";
-        String date4 = "31.06.1997";
-        String date5 = "12.12.1912";
+        List<Student> studentList = generateStudentList();
+        List<Student> filmFaculty = printFilmFacultyStudents(studentList);
```

Comment:
You obtain list of students here, but use it in line 15 (don't use in line 14)
Please, shorten the distance between value declaration and its use to the minimum.